### PR TITLE
SOFTWARE-5538: update github actions to build el9 image

### PR DESCRIPTION
**Pull request by mwestphall**

### CODE REVIEW

File: .github/workflows/build-container.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ jobs:
           OS: ${{ matrix.os }}
         run: |
           PREFIX="output_image=${GITHUB_REPOSITORY}:${SERIES}"
-          if [[ $OS == 'el8' ]]; then
+          if [[ $OS == 'el9' ]]; then
```
Comment (mwestphall) [inline .github/workflows/build-container.yml:28]: This change should change the "no os" tag from el8 to el9. Is that the desired behavior or should it be kept on el8?

Comment (brianhlin) [reply .github/workflows/build-container.yml:28]:
```suggestion
          if [[ $OS == 'el8' ]]; then
```

Hrm yeah, I don't think we can pull the trigger on this OS update just yet.

**Review by brianhlin — CHANGES_REQUESTED** [review]
One in-line change requested

Could you also update the PR title + commit to include the Jira ticket key?

**Review by brianhlin — APPROVED** [review]
LGTM

**Pull request merged**